### ISSUE TITLE
Redesigned homepage's CoP section img tag: Add alt property and remove the `/`

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3792]
### Overview
On the redesigned homepage's Communities of Practice (CoP) section's images, 
  - we want to add an `alt` property so that the images are Web Content Accessibility Guidelines (WCAG) compliant and
  - we want to remove the `/` at the end of the `img` tag so that `img` tags are used uniformly across our website

### Details
<details>
  <summary>Click here to see a screenshot of the redesigned homepage's Communities of Practice section</summary>
  <img width="774" alt="" src="https://user-images.githubusercontent.com/31293603/210579643-501d6002-f94b-4200-9a2d-4bc840fa1666.png">
</details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] In VS Code or another IDE on your computer, navigate to `_includes/home-page/home-communities.html` and change the following from
  ```
  <img src="{{ community[1].icon }}" />
  ```
  to
  ```
  <img src="{{ community[1].icon }}" alt="">
  ```
- [x] Using developer tools in your browser, on the redesigned homepage,
  - [x] Check the `alt` property shows up for each of the images under the Communities of Practice section. (**Note:** Sometimes when you have `alt=""` in the code, it will show up as `alt` when you view the image using developer tools. This is acceptable.)
  - [x] Check your changes do not break anything on the website in mobile, tablet, and desktop views
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article

**Tech Lead or Merge Team**
- [x] After the pull request associated with this issue is approved and merged, please make sure this issue is closed. If it is not closed by the automation, leave a new comment stating what pr number fixes this issue and then close this issue as completed.

### Resources/Instructions
- [How to work off of a feature branch wiki page](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [_includes/home-page/home-communities.html on feature-homepage-launch branch](https://github.com/hackforla/website/blob/feature-homepage-launch/_includes/home-page/home-communities.html)
- The img tag does not need the `/` at the end because we are not using XML or XHTML. See this Stack Overflow answer for more info: https://stackoverflow.com/a/15149657

**Comment by vmaineng** [comment]
i. Availability: (When are you available to work on the issue/answer questions other programmers might have about your issue?)
I am available to work on this issue right now.

ii. ETA: (When do you expect this issue to be completed?)
End of tomorrow after developer's meeting.

**Comment by vmaineng** [comment]
Submitted a PR for this issue.

**Comment by vmaineng** [comment]
Submitted a new PR to be merged with feature-homepage-launch instead of gh-pages

**Comment by MattPereira** [comment]
Hey @JessicaLucindaCheng it looks like you accidentally omitted the self closing `/` in the "to" section of the action items for the `<img />` tag

**Comment by MattPereira** [comment]
- Pull Request: #3883 fixed this issue
